Header header
int16 id
int16 num_covered
int16[] targets_covered